pyproject.toml

```diff
@@ -177,6 +174,14 @@ privacy = [
 project-name = "Temporal Python"
 sidebar-expand-depth = 2
 
+[tool.ruff]
+target-version = "py38"
+extend-exclude = [
+  "temporalio/api",
+  "temporalio/bridge",
+  "tests/worker/workflow_sandbox/testmodules/proto",
```

Comment:
I think these should be included and we should format the generated code too (and our bridge code)

Reply:
Done!